inclusive-design/wecount.inclusivedesign.ca
# feat: configure Netlify CMS for workshops

## Description

This PR migrates workshops from Airtable into an Eleventy file-based collection, and configures a Netlify CMS instance to manage the collection.

## Steps to test

1. Review the Initiatives page on the deploy preview.
2. Check out the branch, run `npm start`, and navigate to http://localhost:3000/admin.

**Expected behavior:**

1. The Initiatives page and individual initiatives are properly displayed.
2. The workshops are editable via the CMS interface with live previews.

## Additional information

See updated [docs](https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/0e0840886c2ca7385fdb2aadc1570ac6c08ae4cc/README.md#content-management-system-cms).

Note that the CMS is set to manage the `dev` branch currently. This will need to be changed once this PR has been tested and is merged to `main`.

## Related issues

Not applicable.
